docs/animations/Blur.md

```diff
@@ -128,5 +128,5 @@ Await MyUIElement.Blur(value:=5, duration:=2500, delay:=250).StartAsync()    ' B
 
 ## Related Topics
 
-- [AnimationSet Class](/windows/communitytoolkit/animations/animationset)
+- [AnimationSet Class](./animationset.md)

```

Comment:
Not sure if this'll break our sample app again, this happened before we shipped 7.0 which I hadn't realized. I spent time the weekend before we shipped fixing this, but not sure how well it'll handle the relative path. We'll have to test again.

Is `./` a common thing, why wouldn't it just be `animationset.md` without the `./` if it's in the same path?

Reply:
Both work.  We had early requests for the format to be `./` so that's what the script does.  You can leave it or update it.  DocFx/OPS accept both.